src/modules/auth/controllers/het-archief.controller.ts

```diff
@@ -163,6 +164,14 @@ export class HetArchiefController {
 				}
 			}
 
+			// CP_ADMIN: Link the user to the maintainer
+			if (userGroup === Group.CP_ADMIN) {
```

Comment:
also for kiosk users
or do you want to do that in a separate PR?

Reply:
Updated in this PR